geode-core/src/main/java/org/apache/geode/internal/monitoring/ThreadsMonitoringProcess.java

```diff
@@ -33,73 +31,86 @@
 
 public class ThreadsMonitoringProcess extends TimerTask {
 
-  private final ThreadsMonitoring threadsMonitoring;
-  private ResourceManagerStats resourceManagerStats = null;
   private static final Logger logger = LogService.getLogger();
+
+  private final ThreadsMonitoring threadsMonitoring;
   private final int timeLimit;
   private final InternalDistributedSystem internalDistributedSystem;
 
-  private final Properties nonDefault = new Properties();
-  private final DistributionConfigImpl distributionConfigImpl =
-      new DistributionConfigImpl(nonDefault);
+  private ResourceManagerStats resourceManagerStats = null;
 
   protected ThreadsMonitoringProcess(ThreadsMonitoring tMonitoring,
-      InternalDistributedSystem iDistributedSystem) {
-    this.timeLimit = this.distributionConfigImpl.getThreadMonitorTimeLimit();
+      InternalDistributedSystem iDistributedSystem, int timeLimit) {
```

Comment:
What unit is timeLimit? I recommend either adding a TimeUnit parameter or renaming timeLimit to include Millis or something as a prefix.

Reply:
I changed the name to timeLimitMillis. This unit is determined by an external geode property so it will always be milliseconds